### ISSUE TITLE
New Library/SDK Enhancement/Feature request: ledgerhq/hw-app-xmr node app

### ISSUE THREAD

**Issue by dotnetspec** [LedgerHQ/app-monero#136]
I'm working on a bounty to build a _mobile web based_ app for the [Monero Haveno project.](https://bounties.monero.social/posts/128/101-211m-mvp-tor-based-mobile-web-application-to-interact-with-ledger-nano-x-and-haveno).

[Version 0.0.18 can be viewed here.](https://haveno-web-dev.netlify.app/)

Using` "@ledgerhq/hw-transport-webusb" `the LNS/LNX in a Chrome based mobile browser (Android) successfully connects. HID is not currently supported in mobile web browsers and I do not believe this is likely to change in the short term.

However, webusb only enables a connection. To interact with the XMR wallet I need an equivalent of 
`@ledgerhq/hw-app-btc`
e.g. `@ledgerhq/hw-app-xmr`

I understand there are requirements specific to Monero that potentially make this more technically challenging than e.g. for Bitcoin. However, the provision of an hw-app-xmr would considerably extend the use cases for Ledger devices in the privacy space.

Any feedback you can give would be much appreciated ...

**Comment by FocuzJS** [comment]
Agreed. Bluetooth support for the monero app on ledger would enable alot in the space of mobile integrations, especially for iOS.
